packages/trader/src/Modules/Trading/Components/Form/TradeParams/barrier.jsx

```diff
@@ -13,14 +14,26 @@ const Barrier = ({
     barrier_count,
     barrier_pipsize,
     current_focus,
+    proposal_info,
     duration_unit,
     is_minimized,
     is_absolute_only,
     onChange,
     setCurrentFocus,
     validation_errors,
 }) => {
+    const contract_type_info =
+        (proposal_info?.CALL?.spot && proposal_info?.CALL) ||
+        proposal_info?.PUT ||
+        (proposal_info?.NOTOUCH?.spot && proposal_info?.NOTOUCH) ||
+        proposal_info?.ONETOUCH;
+    const current_price = contract_type_info?.spot || '';
+    const barrier_price = contract_type_info?.barrier || '';
     const barrier_title = barrier_count === 1 ? localize('Barrier') : localize('Barriers');
+    const [show_modal, setShowModal] = React.useState(false);
```

Comment:
@kate-deriv isn't it better to move this line before defining any other variable in order to better readability?

Reply:
Sure, I'll lift it up👍🏻 But I cant't move line 34 as it's using variable, which is define above